src/legacy/core_plugins/tagcloud/public/feedback_message.js

```diff
@@ -28,21 +29,35 @@ export class FeedbackMessage extends Component {
   }
 
   render() {
+    if (!this.state.shouldShowTruncate && !this.state.shouldShowIncomplete) {
+      return '';
+    }
+
     return (
-      <div className="tgcVisFeedback" >
-        <div className="tgcVisFeedback__message" style={{ display: this.state.shouldShowTruncate ? 'block' : 'none' }}>
-          <FormattedMessage
-            id="tagCloud.feedbackMessage.truncatedTagsDescription"
-            defaultMessage="The number of tags has been truncated to avoid long draw times."
-          />
-        </div>
-        <div className="tgcVisFeedback__message" style={{ display: this.state.shouldShowIncomplete ? 'block' : 'none' }}>
-          <FormattedMessage
-            id="tagCloud.feedbackMessage.tooSmallContainerDescription"
-            defaultMessage="The container is too small to display the entire cloud. Tags might be cropped or omitted."
-          />
-        </div>
-      </div>
+      <EuiIconTip
+        type="alert"
+        color="warning"
```

Comment:
Alternating between just using `text` as the color here. It's less visually distracting and potentially more readible. On the fence.

Reply:
But at the same time in needs to be noticeable because it's saying "This doesn't actually represent all the possible words", so in essence the vis is then not so reliable.

Reply:
Yep. On the fence. Go with your gut.